src/symbols.h

```diff
@@ -15,14 +15,28 @@ class Symbols {
     static Mutex _parse_lock;
     static bool _have_kernel_symbols;
     static bool _libs_limit_reported;
+    static const void* _main_phdr;
+    static const char* _ld_base;
```

Comment:
Those are Linux-specific and not part of Symbols public interface.
Can be just static variables in `symbols_linux.cpp`.

Reply:
8f57df7def2cb55da274cd498c91b7d6aa228de0